tracer/src/Datadog.Trace/Util/RuntimeId.cs

```diff
@@ -13,22 +13,23 @@ namespace Datadog.Trace.Util
 {
     internal static class RuntimeId
     {
-#if NETFRAMEWORK
-        private const string NativeLoaderLibNameX86 = "Datadog.AutoInstrumentation.NativeLoader.x86.dll";
-        private const string NativeLoaderLibNameX64 = "Datadog.AutoInstrumentation.NativeLoader.x64.dll";
-#else
-        private const string NativeLoaderLibNameX86 = "Datadog.AutoInstrumentation.NativeLoader.x86";
-        private const string NativeLoaderLibNameX64 = "Datadog.AutoInstrumentation.NativeLoader.x64";
-#endif
-
         private static readonly IDatadogLogger Log = DatadogLogging.GetLoggerFor(typeof(RuntimeId));
         private static string _runtimeId;
 
+        private static bool IsNativeLoaderAvailable
+        {
+            get
+            {
+                var fd = FrameworkDescription.Create();
```

Comment:
Is there a reason to not use the static (cached) `FrameworkDescription.Instance` here?
```suggestion
                var fd = FrameworkDescription.Instance;
```

Reply:
I did not know, I copy/pasted it from libddwaf usage.